os/media/Kconfig

```diff
@@ -0,0 +1,34 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config MEDIA
+	bool "Media Support"
+	default n
+	---help---
```

Comment:
There is no dependancy with audio. Will you allow enabling media without audio?

Reply:
Dependency added